mmv1/products/notebooks/api.yaml

```diff
@@ -245,6 +254,31 @@ objects:
             required: true
             description: |
               Count of cores of this accelerator.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'shieldedInstanceConfig'
+        description: |
+            A set of Shielded Instance options. Check [Images using supported Shielded VM features]
+            Not all combinations are valid
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enableIntegrityMonitoring'
+            description: |
+              Defines whether the instance has integrity monitoring enabled. Enables monitoring and attestation of the
+              boot integrity of the instance. The attestation is performed against the integrity policy baseline.
+              This baseline is initially derived from the implicitly trusted boot image when the instance is created.
+              Enabled by default.
```

Comment:
Should we set a provider-side default to `true`?

Reply:
ack